ubuntui/widgets/input.py

```diff
@@ -179,15 +179,22 @@ def set_default(self, item, state=False):
 
 class YesNo(Selector):
     """ Yes/No selector
+
+    Note: The :prop:`.value` will be either ``True`` or ``False`` rather than
```

Comment:
@johnsca why are we doing True/False for .value() instead of Yes/No?

Reply:
Not really sure, probably just for historical reasons.  I think it would be good to clean this up but I'm not certain where all this is used.  If it's only used in conjure-up (as I suspect), then we could just remove the special handling here and put that into conjure-up where it belongs.